data-raw/standard-format-data-prep/covariate_stock_recruit.Rmd

```diff
@@ -53,10 +88,45 @@ ggplot(butte_dd_sum, aes(wyear, totalddays)) +
 ```
 
 
-## Day of year
+## Day of year when 7DADM drops below 13 C
+#Butte Creek
+```{r}
+#Filtering Butte data 
+butte_std_temp <- read.csv(here::here("data-raw", "standard-format-data-prep", "temp_data", "standard_temperature.csv")) |>
```

Comment:
instead of adding in the file path again can just do:

`butte_std_temp <- strd_temp |>
  filter(stream == "butte creek")`